### ISSUE TITLE
Universal Storage/ Cryo Tanks Not Working

### ISSUE THREAD

**Issue by wolfie1897** [linuxgurugamer/universal-storage-2#17]
Hello, when I use US2 and Cryo Tanks and select a CT fuel for the US2 Fuel Tank, it does not show the fuel in the part window, and appears to be completely empty when a craft is launched

To replicate: On a save with US2 and CT, select the fuel LH2 on the US2 Fuel Tank

**Comment by wsinger** [comment]
This is because of the balance patch in the Vandist Directory not having values for cryo tanks. Deleting this worked for me

**Comment by linuxgurugamer** [comment]
Did you consider adding values for cryo tanks?

**Comment by wsinger** [comment]
Yeah I'm currently playing around with some of the patch files related to cryo tanks. Currently trying to get the Storage cores to support cryo tank fuels in their central payload (for some reason they arean't rendering outside the VAB).

**Comment by wsinger** [comment]
Why are the balance and recolor patches applied as patches rather than edits to the original cfg files?

**Comment by EndsM** [comment]
So the patching logic is reassign the value, instead append the config to each property of the object?

**Comment by linuxgurugamer** [comment]
Patches are applied with ModuleManager.  They are applied according to how they are defined (ie:  if something is required ), but after that they are applied sequentially.  There are rules which MM has, which you can say a patch is applied before, or after other specified mods, or at the end.
The rules and syntax is fully explained in the MM wiki here:  https://github.com/sarbian/ModuleManager/wiki

Patches can be applied, merged and/or replace existing code

**Comment by linuxgurugamer** [comment]
> Yeah I'm currently playing around with some of the patch files related to cryo tanks. Currently trying to get the Storage cores to support cryo tank fuels in their central payload (for some reason they arean't rendering outside the VAB).

Let me know if you need some help with that.  I'm kind of busy these days, but should be able to help if you need it

**Comment by linuxgurugamer** [comment]
You can see the final result of ALL the patching by looking at the file:  GameData/ModuleManager.ConfigCache.
You can see the patch log in Logs/ModuleManager
